{
 "about": {
  "channels": [
   "https://conda.anaconda.org/conda-forge"
  ],
  "conda_build_version": "3.25.0",
  "conda_version": "23.3.1",
  "env_vars": {
   "CIO_TEST": "<not set>"
  },
  "extra": {
   "copy_test_source_files": true,
   "final": true,
   "recipe-maintainers": [
    "rxm7706"
   ]
  },
  "home": "https://github.com/microsoft/responsible-ai-toolbox",
  "identifiers": [],
  "keywords": [],
  "license": "MIT",
  "license_file": "LICENSE",
  "root_pkgs": [
   "libsqlite 3.42.0 h2797004_0",
   "tzdata 2023c h71feb2d_0",
   "brotli-python 1.0.9 py310hd8f1fbe_9",
   "toolz 0.12.0 pyhd8ed1ab_0",
   "tqdm 4.65.0 pyhd8ed1ab_1",
   "libgomp 13.1.0 he5830b7_0",
   "ruamel.yaml 0.17.32 py310h2372a71_0",
   "libexpat 2.5.0 hcb278e6_1",
   "libdeflate 1.18 h0b41bf4_0",
   "bzip2 1.0.8 h7f98852_4",
   "pycparser 2.21 pyhd8ed1ab_0",
   "patchelf 0.17.2 h58526e2_0",
   "freetype 2.12.1 hca18f0e_1",
   "prompt-toolkit 3.0.39 pyha770c72_0",
   "_openmp_mutex 4.5 2_gnu",
   "six 1.16.0 pyh6c4a22f_0",
   "filelock 3.12.2 pyhd8ed1ab_0",
   "conda-pack 0.7.0 pyh6c4a22f_0",
   "libffi 3.4.2 h7f98852_5",
   "libstdcxx-ng 13.1.0 hfd8a6a1_0",
   "tornado 6.3.2 py310h2372a71_0",
   "requests-toolbelt 1.0.0 pyhd8ed1ab_0",
   "anyio 3.7.1 pyhd8ed1ab_0",
   "boltons 23.0.0 pyhd8ed1ab_0",
   "krb5 1.21.1 h659d440_0",
   "git 2.41.0 pl5321h86e50cf_0",
   "libjpeg-turbo 2.1.5.1 h0b41bf4_0",
   "zipp 3.16.2 pyhd8ed1ab_0",
   "jsonpatch 1.32 pyhd8ed1ab_0",
   "anaconda-project 0.11.1 pyhd8ed1ab_0",
   "libmambapy 1.4.2 py310h1428755_0",
   "pip 23.2.1 pyhd8ed1ab_0",
   "conda-build 3.25.0 py310hff52083_0",
   "xz 5.2.6 h166bdaf_0",
   "lzo 2.10 h516909a_1000",
   "libssh2 1.11.0 h0841786_0",
   "colorama 0.4.6 pyhd8ed1ab_0",
   "jsonpointer 2.0 py_0",
   "packaging 23.1 pyhd8ed1ab_0",
   "typing-extensions 4.7.1 hd8ed1ab_0",
   "ld_impl_linux-64 2.40 h41732ed_0",
   "tk 8.6.12 h27826a3_0",
   "mdurl 0.1.0 pyhd8ed1ab_0",
   "jupyter_core 5.3.1 py310hff52083_0",
   "exceptiongroup 1.1.2 pyhd8ed1ab_0",
   "backports.functools_lru_cache 1.6.5 pyhd8ed1ab_0",
   "jsonschema 4.18.6 pyhd8ed1ab_0",
   "urllib3 1.26.15 pyhd8ed1ab_0",
   "libmamba 1.4.2 hcea66bb_0",
   "libxml2 2.11.4 h0d562d8_0",
   "conda-package-handling 2.2.0 pyh38be061_0",
   "traitlets 5.9.0 pyhd8ed1ab_0",
   "pybind11-abi 4 hd8ed1ab_3",
   "more-itertools 10.1.0 pyhd8ed1ab_0",
   "jsonschema-specifications 2023.7.1 pyhd8ed1ab_0",
   "chardet 5.1.0 py310hff52083_0",
   "fmt 9.1.0 h924138e_0",
   "pytz 2023.3 pyhd8ed1ab_0",
   "pillow 10.0.0 py310h582fbeb_0",
   "glob2 0.7 py_0",
   "c-ares 1.19.1 hd590300_0",
   "libedit 3.1.20191231 he28a2e2_2",
   "yaml 0.2.5 h7f98852_2",
   "openjpeg 2.5.0 hfec8fc6_2",
   "attrs 23.1.0 pyh71513ae_1",
   "xorg-libxdmcp 1.1.3 h7f98852_0",
   "psutil 5.9.5 py310h1fa729e_0",
   "wheel 0.41.1 pyhd8ed1ab_0",
   "tomli 2.0.1 pyhd8ed1ab_0",
   "libzlib 1.2.13 hd590300_5",
   "zstd 1.5.2 hfc55251_7",
   "python_abi 3.10 3_cp310",
   "libxcb 1.15 h0b41bf4_0",
   "markupsafe 2.1.3 py310h2372a71_0",
   "yaml-cpp 0.7.0 h27087fc_2",
   "python 3.10.12 hd12c33a_0_cpython",
   "defusedxml 0.7.1 pyhd8ed1ab_0",
   "pkgutil-resolve-name 1.3.10 pyhd8ed1ab_0",
   "ruamel_yaml 0.15.80 py310h5764c6d_1008",
   "pkginfo 1.9.6 pyhd8ed1ab_0",
   "setuptools 68.0.0 pyhd8ed1ab_0",
   "libtiff 4.5.1 h8b53f26_0",
   "json5 0.9.14 pyhd8ed1ab_0",
   "python-dateutil 2.8.2 pyhd8ed1ab_0",
   "lcms2 2.15 haa2dc70_1",
   "libiconv 1.17 h166bdaf_0",
   "su-exec 0.2 h166bdaf_1003",
   "charset-normalizer 3.2.0 pyhd8ed1ab_0",
   "gettext 0.21.1 h27087fc_0",
   "libsolv 0.7.24 hfc55251_1",
   "libcurl 8.2.1 hca28451_0",
   "keyutils 1.6.1 h166bdaf_0",
   "patch 2.7.6 h7f98852_1002",
   "lz4-c 1.9.4 hcb278e6_0",
   "lerc 4.0.0 h27087fc_0",
   "tini 0.19.0 h166bdaf_1",
   "typing_extensions 4.7.1 pyha770c72_0",
   "libgcc-ng 13.1.0 he5830b7_0",
   "beautifulsoup4 4.12.2 pyha770c72_0",
   "curl 8.2.1 hca28451_0",
   "pygments 2.15.1 pyhd8ed1ab_0",
   "watchgod 0.8.2 pyhd8ed1ab_0",
   "joblib 1.3.0 pyhd8ed1ab_1",
   "cffi 1.15.1 py310h255011f_3",
   "click 8.1.6 unix_pyh707e725_0",
   "ripgrep 13.0.0 h2f28480_2",
   "libnsl 2.0.0 h7f98852_0",
   "perl 5.32.1 4_hd590300_perl5",
   "markdown-it-py 3.0.0 pyhd8ed1ab_0",
   "py-lief 0.12.3 py310hd8f1fbe_0",
   "conda-index 0.2.3 pyhd8ed1ab_0",
   "nbformat 5.9.2 pyhd8ed1ab_0",
   "_libgcc_mutex 0.1 conda_forge",
   "reproc-cpp 14.2.4 hcb278e6_0",
   "libuuid 2.38.1 h0b41bf4_0",
   "libnghttp2 1.52.0 h61bc06f_0",
   "libwebp-base 1.3.1 hd590300_0",
   "boa 0.15.1 pyhd8ed1ab_0",
   "ca-certificates 2023.7.22 hbcca054_0",
   "liblief 0.12.3 h27087fc_0",
   "pysocks 1.7.1 pyha2e5f31_6",
   "openssl 3.1.2 hd590300_0",
   "backports 1.0 pyhd8ed1ab_3",
   "libpng 1.6.39 h753d276_0",
   "rpds-py 0.9.2 py310hcb5633a_0",
   "icu 72.1 hcb278e6_0",
   "pcre2 10.40 hc3806b6_0",
   "reproc 14.2.4 h0b41bf4_0",
   "zstandard 0.19.0 py310h1275a96_2",
   "libev 4.33 h516909a_1",
   "xorg-libxau 1.0.11 hd590300_0",
   "libarchive 3.6.2 h039dbb9_1",
   "importlib_resources 6.0.0 pyhd8ed1ab_1",
   "soupsieve 2.3.2.post1 pyhd8ed1ab_0",
   "conda-package-streaming 0.9.0 pyhd8ed1ab_0",
   "wcwidth 0.2.6 pyhd8ed1ab_0",
   "jinja2 3.1.2 pyhd8ed1ab_1",
   "conda 23.3.1 py310hff52083_0",
   "sniffio 1.3.0 pyhd8ed1ab_0",
   "mamba 1.4.2 py310h51d5547_0",
   "referencing 0.30.1 pyhd8ed1ab_0",
   "pthread-stubs 0.4 h36c2ea0_1001",
   "ruamel.yaml.clib 0.2.7 py310h1fa729e_1",
   "anaconda-client 1.12.0 pyhd8ed1ab_1",
   "brotlipy 0.7.0 py310h5764c6d_1005",
   "prompt_toolkit 3.0.39 hd8ed1ab_0",
   "pluggy 1.2.0 pyhd8ed1ab_0",
   "rich 13.5.1 pyhd8ed1ab_0",
   "python-libarchive-c 5.0 py310hff52083_1",
   "certifi 2023.7.22 pyhd8ed1ab_0",
   "platformdirs 3.10.0 pyhd8ed1ab_0",
   "dataclasses 0.8 pyhc8e2a94_3",
   "pyyaml 6.0 py310h5764c6d_5",
   "pycosat 0.6.4 py310h5764c6d_1",
   "clyent 1.2.2 py_1",
   "idna 3.4 pyhd8ed1ab_0",
   "ncurses 6.4 hcb278e6_0",
   "requests 2.31.0 pyhd8ed1ab_0",
   "python-fastjsonschema 2.18.0 pyhd8ed1ab_0",
   "pyopenssl 23.2.0 pyhd8ed1ab_1",
   "readline 8.2 h8228510_1",
   "cryptography 41.0.2 py310h75e40e8_0",
   "conda-forge-ci-setup 3.32.5 py310h7a2d8a0_100",
   "jq 1.6 h36c2ea0_1000",
   "oras-py 0.1.14 pyhd8ed1ab_0",
   "conda-env 2.6.0 1",
   "conda-forge-metadata 0.5.2 pyhd8ed1ab_0",
   "oniguruma 6.9.8 h166bdaf_0",
   "conda-oci-mirror 0.1.0 pyhd8ed1ab_0",
   "shyaml 0.6.2 pyhd3deb0d_0"
  ],
  "summary": "SDK API to explain models, generate counterfactual examples, analyze causal effects and analyze errors in Machine Learning models.",
  "tags": []
 },
 "conda_build_config": {
  "CI": "azure",
  "c_compiler": "gcc",
  "cdt_name": "cos6",
  "channel_sources": "conda-forge",
  "channel_targets": "conda-forge main",
  "cpu_optimization_target": "nocona",
  "cran_mirror": "https://cran.r-project.org",
  "cxx_compiler": "gxx",
  "docker_image": "quay.io/condaforge/linux-anvil-cos7-x86_64",
  "extend_keys": [
   "extend_keys",
   "ignore_build_only_deps",
   "ignore_version",
   "pin_run_as_build"
  ],
  "fortran_compiler": "gfortran",
  "ignore_build_only_deps": [
   "python",
   "numpy"
  ],
  "lua": "5",
  "numpy": "1.22",
  "perl": "5.26.2",
  "pin_run_as_build": {
   "python": {
    "max_pin": "x.x",
    "min_pin": "x.x"
   },
   "r-base": {
    "max_pin": "x.x",
    "min_pin": "x.x"
   }
  },
  "python": "3.10",
  "r_base": "3.5",
  "target_platform": "linux-64"
 },
 "conda_pkg_format": "2",
 "files": [
  "site-packages/responsibleai-0.29.0.dist-info/INSTALLER",
  "site-packages/responsibleai-0.29.0.dist-info/LICENSE",
  "site-packages/responsibleai-0.29.0.dist-info/METADATA",
  "site-packages/responsibleai-0.29.0.dist-info/RECORD",
  "site-packages/responsibleai-0.29.0.dist-info/REQUESTED",
  "site-packages/responsibleai-0.29.0.dist-info/WHEEL",
  "site-packages/responsibleai-0.29.0.dist-info/direct_url.json",
  "site-packages/responsibleai/__init__.py",
  "site-packages/responsibleai/__version__.py",
  "site-packages/responsibleai/_config/__init__.py",
  "site-packages/responsibleai/_config/base_config.py",
  "site-packages/responsibleai/_data_validations.py",
  "site-packages/responsibleai/_interfaces.py",
  "site-packages/responsibleai/_internal/__init__.py",
  "site-packages/responsibleai/_internal/_forecasting_wrappers.py",
  "site-packages/responsibleai/_internal/constants.py",
  "site-packages/responsibleai/_tools/__init__.py",
  "site-packages/responsibleai/_tools/causal/__init__.py",
  "site-packages/responsibleai/_tools/causal/causal_config.py",
  "site-packages/responsibleai/_tools/causal/causal_constants.py",
  "site-packages/responsibleai/_tools/causal/causal_result.py",
  "site-packages/responsibleai/_tools/causal/dashboard_schemas/__init__.py",
  "site-packages/responsibleai/_tools/causal/dashboard_schemas/schema_0.1.0.json",
  "site-packages/responsibleai/_tools/counterfactual/__init__.py",
  "site-packages/responsibleai/_tools/counterfactual/dashboard_schemas/__init__.py",
  "site-packages/responsibleai/_tools/counterfactual/dashboard_schemas/counterfactual_examples_output_v1.0.json",
  "site-packages/responsibleai/_tools/counterfactual/dashboard_schemas/counterfactual_examples_output_v2.0.json",
  "site-packages/responsibleai/_tools/error_analysis/__init__.py",
  "site-packages/responsibleai/_tools/error_analysis/dashboard_schemas/__init__.py",
  "site-packages/responsibleai/_tools/error_analysis/dashboard_schemas/error_analysis_output_v0.0.json",
  "site-packages/responsibleai/_tools/shared/__init__.py",
  "site-packages/responsibleai/_tools/shared/attribute_serialization.py",
  "site-packages/responsibleai/_tools/shared/base_result.py",
  "site-packages/responsibleai/_tools/shared/state_directory_management.py",
  "site-packages/responsibleai/_tools/shared/versions.py",
  "site-packages/responsibleai/databalanceanalysis/__init__.py",
  "site-packages/responsibleai/databalanceanalysis/aggregate_balance_measures.py",
  "site-packages/responsibleai/databalanceanalysis/balance_measures.py",
  "site-packages/responsibleai/databalanceanalysis/constants.py",
  "site-packages/responsibleai/databalanceanalysis/data_balance_utils.py",
  "site-packages/responsibleai/databalanceanalysis/distribution_balance_measures.py",
  "site-packages/responsibleai/databalanceanalysis/feature_balance_measures.py",
  "site-packages/responsibleai/exceptions.py",
  "site-packages/responsibleai/feature_metadata.py",
  "site-packages/responsibleai/managers/__init__.py",
  "site-packages/responsibleai/managers/base_manager.py",
  "site-packages/responsibleai/managers/causal_manager.py",
  "site-packages/responsibleai/managers/counterfactual_manager.py",
  "site-packages/responsibleai/managers/data_balance_manager.py",
  "site-packages/responsibleai/managers/error_analysis_manager.py",
  "site-packages/responsibleai/managers/explainer_manager.py",
  "site-packages/responsibleai/modelanalysis/__init__.py",
  "site-packages/responsibleai/modelanalysis/constants/__init__.py",
  "site-packages/responsibleai/modelanalysis/model_analysis.py",
  "site-packages/responsibleai/rai_insights/__init__.py",
  "site-packages/responsibleai/rai_insights/rai_base_insights.py",
  "site-packages/responsibleai/rai_insights/rai_insights.py",
  "site-packages/responsibleai/serialization_utilities.py",
  "site-packages/responsibleai/utils.py",
  "site-packages/tests/__init__.py",
  "site-packages/tests/causal/__init__.py",
  "site-packages/tests/causal/conftest.py",
  "site-packages/tests/causal/test_causal_constants.py",
  "site-packages/tests/causal/test_causal_different_data_types.py",
  "site-packages/tests/causal/test_causal_general.py",
  "site-packages/tests/causal/test_causal_manager.py",
  "site-packages/tests/causal/test_causal_result.py",
  "site-packages/tests/causal/test_causal_result_versioning.py",
  "site-packages/tests/causal_manager_validator.py",
  "site-packages/tests/common_utils.py",
  "site-packages/tests/conftest.py",
  "site-packages/tests/counterfactual/__init__.py",
  "site-packages/tests/counterfactual/test_counterfactual_advanced_features.py",
  "site-packages/tests/counterfactual_manager_validator.py",
  "site-packages/tests/databalanceanalysis/__init__.py",
  "site-packages/tests/databalanceanalysis/conftest.py",
  "site-packages/tests/databalanceanalysis/test_aggregate_balance_measures.py",
  "site-packages/tests/databalanceanalysis/test_data_balance_helper.py",
  "site-packages/tests/databalanceanalysis/test_data_balance_manager.py",
  "site-packages/tests/databalanceanalysis/test_distribution_balance_measures.py",
  "site-packages/tests/databalanceanalysis/test_feature_balance_measures.py",
  "site-packages/tests/error_analysis_validator.py",
  "site-packages/tests/explainer/__init__.py",
  "site-packages/tests/explainer/test_explainer_manager.py",
  "site-packages/tests/explainer_manager_validator.py",
  "site-packages/tests/model_analysis/__init__.py",
  "site-packages/tests/model_analysis/test_model_analysis.py",
  "site-packages/tests/model_analysis/test_model_analysis_validations.py",
  "site-packages/tests/rai_insights/__init__.py",
  "site-packages/tests/rai_insights/test_rai_insights.py",
  "site-packages/tests/rai_insights/test_rai_insights_cohort_filter.py",
  "site-packages/tests/rai_insights/test_rai_insights_data_size_scenarios.py",
  "site-packages/tests/rai_insights/test_rai_insights_get_feature_ranges.py",
  "site-packages/tests/rai_insights/test_rai_insights_save_and_load_scenarios.py",
  "site-packages/tests/rai_insights/test_rai_insights_validations.py",
  "site-packages/tests/test_dependencies.py",
  "site-packages/tests/test_exceptions.py",
  "site-packages/tests/test_feature_metadata.py",
  "site-packages/tests/test_model_serializer.py",
  "site-packages/tests/test_serialization_utilities.py",
  "site-packages/tests/tools/__init__.py",
  "site-packages/tests/tools/shared/__init__.py",
  "site-packages/tests/tools/shared/test_attribute_serialization.py",
  "site-packages/tests/tools/shared/test_state_directory_management.py",
  "site-packages/tests/tools/shared/test_versions.py"
 ],
 "index": {
  "arch": null,
  "build": "pyhd8ed1ab_0",
  "build_number": 0,
  "depends": [
   "dice-ml >=0.10,<0.11",
   "econml >=0.14.1",
   "erroranalysis >=0.4.4",
   "interpret-community >=0.29.0",
   "ipykernel <=6.8.0",
   "jsonschema",
   "lightgbm >=2.0.11",
   "markupsafe <=2.1.2",
   "ml-wrappers",
   "networkx <=2.5",
   "numba <=0.55.2",
   "numpy >=1.17.2,<1.24.0",
   "pandas >=0.25.1,<2.0.0",
   "python >=3.6",
   "raiutils >=0.4.0",
   "scikit-learn >=0.22.1,<1.1",
   "scipy >=1.4.1",
   "semver >=2.13.0,<2.14.dev0",
   "statsmodels <0.14.0"
  ],
  "license": "MIT",
  "name": "responsibleai",
  "noarch": "python",
  "platform": null,
  "subdir": "noarch",
  "timestamp": 1691485272711,
  "version": "0.29.0"
 },
 "metadata_version": 1,
 "name": "responsibleai",
 "raw_recipe": "{% set name = \"responsibleai\" %}\n{% set version = \"0.29.0\" %}\n\npackage:\n  name: {{ name|lower }}\n  version: {{ version }}\n\nsource:\n  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/responsibleai-{{ version }}.tar.gz\n  sha256: 0008893b296b95bac74f35da0931ff7fab2fe39781cd96178cfa3cfec2e62fd3\n\nbuild:\n  noarch: python\n  script: {{ PYTHON }} -m pip install . -vv\n  number: 0\n\nrequirements:\n  host:\n    - python >=3.6\n    - pip\n  run:\n    - python >=3.6\n    - dice-ml >=0.10,<0.11\n    - econml >=0.14.1\n    - statsmodels <0.14.0\n    - jsonschema\n    - erroranalysis >=0.4.4\n    - interpret-community >=0.29.0\n    - lightgbm >=2.0.11\n    - numpy >=1.17.2,<1.24.0\n    - numba <=0.55.2\n    - pandas >=0.25.1,<2.0.0\n    - scikit-learn >=0.22.1,<1.1\n    - scipy >=1.4.1\n    - semver >=2.13.0,<2.14.dev0\n    - ml-wrappers\n    - networkx <=2.5\n    - ipykernel <=6.8.0\n    - markupsafe <=2.1.2\n    - raiutils >=0.4.0\n\ntest:\n  imports:\n    - responsibleai\n  commands:\n    - pip check\n  requires:\n    - pip\n\nabout:\n  home: https://github.com/microsoft/responsible-ai-toolbox\n  summary: SDK API to explain models, generate counterfactual examples, analyze causal effects and analyze errors in Machine Learning models.\n  license: MIT\n  license_file: LICENSE\n\nextra:\n  recipe-maintainers:\n    - rxm7706\n",
 "rendered_recipe": {
  "about": {
   "home": "https://github.com/microsoft/responsible-ai-toolbox",
   "license": "MIT",
   "license_file": "LICENSE",
   "summary": "SDK API to explain models, generate counterfactual examples, analyze causal effects and analyze errors in Machine Learning models."
  },
  "build": {
   "noarch": "python",
   "number": "0",
   "script": "/home/conda/feedstock_root/build_artifacts/responsibleai_1691485060805/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/bin/python -m pip install . -vv",
   "string": "pyhd8ed1ab_0"
  },
  "extra": {
   "copy_test_source_files": true,
   "final": true,
   "recipe-maintainers": [
    "rxm7706"
   ]
  },
  "package": {
   "name": "responsibleai",
   "version": "0.29.0"
  },
  "requirements": {
   "host": [
    "_libgcc_mutex 0.1 conda_forge",
    "_openmp_mutex 4.5 2_gnu",
    "bzip2 1.0.8 h7f98852_4",
    "ca-certificates 2023.7.22 hbcca054_0",
    "ld_impl_linux-64 2.40 h41732ed_0",
    "libexpat 2.5.0 hcb278e6_1",
    "libffi 3.4.2 h7f98852_5",
    "libgcc-ng 13.1.0 he5830b7_0",
    "libgomp 13.1.0 he5830b7_0",
    "libnsl 2.0.0 h7f98852_0",
    "libsqlite 3.42.0 h2797004_0",
    "libuuid 2.38.1 h0b41bf4_0",
    "libzlib 1.2.13 hd590300_5",
    "ncurses 6.4 hcb278e6_0",
    "openssl 3.1.2 hd590300_0",
    "pip 23.2.1 pyhd8ed1ab_0",
    "python 3.11.4 hab00c5b_0_cpython",
    "readline 8.2 h8228510_1",
    "setuptools 68.0.0 pyhd8ed1ab_0",
    "tk 8.6.12 h27826a3_0",
    "tzdata 2023c h71feb2d_0",
    "wheel 0.41.1 pyhd8ed1ab_0",
    "xz 5.2.6 h166bdaf_0"
   ],
   "run": [
    "dice-ml >=0.10,<0.11",
    "econml >=0.14.1",
    "erroranalysis >=0.4.4",
    "interpret-community >=0.29.0",
    "ipykernel <=6.8.0",
    "jsonschema",
    "lightgbm >=2.0.11",
    "markupsafe <=2.1.2",
    "ml-wrappers",
    "networkx <=2.5",
    "numba <=0.55.2",
    "numpy >=1.17.2,<1.24.0",
    "pandas >=0.25.1,<2.0.0",
    "python >=3.6",
    "raiutils >=0.4.0",
    "scikit-learn >=0.22.1,<1.1",
    "scipy >=1.4.1",
    "semver >=2.13.0,<2.14.dev0",
    "statsmodels <0.14.0"
   ]
  },
  "source": {
   "sha256": "0008893b296b95bac74f35da0931ff7fab2fe39781cd96178cfa3cfec2e62fd3",
   "url": "https://pypi.io/packages/source/r/responsibleai/responsibleai-0.29.0.tar.gz"
  },
  "test": {
   "commands": [
    "pip check"
   ],
   "imports": [
    "responsibleai"
   ],
   "requires": [
    "pip"
   ]
  }
 },
 "version": "0.29.0"
}